src/events/Ready.js

```diff
@@ -11,19 +11,28 @@ class Ready extends KongouEvent {
 
     async run() {
         this.client.logger.debug(`${this.client.user.username}`, `Ready! Serving ${this.client.guilds.cache.size} guild(s) with ${this.client.users.cache.size} user(s)`);
-        if (!this.interval) {
-            await this.client.user.setActivity('It\'s the English-born returnee, Kongou! Nice to meet you!');
-            const statuses =  [
-                'Admiral! You\'ve got mail! Love Letters aren\'t allowed!',
-                'Always got to make time for tea-time!',
-                'Burning... Love!'
+        //if (!this.interval) {
```

Comment:
you can remove the commented out code to reduce the amount of code in the file

Reply:
> you can remove the commented out code to reduce the amount of code in the file

Yup i am done with it